### PR TITLE
[GSoC] Modify docs of is_stable method in TransferFunction class

### PR DESCRIPTION
**Pull request by namannimmo10**
#### Brief description of what is fixed or changed
Adds the comment in an appropriate position.

#### Other comments

Screenshot from 2020-08-28 12-44-30

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by Sc0rpi0n101** [comment]
@namannimmo10 does it render fine now?

**Comment by namannimmo10** [comment]
I haven't checked it yet but looks like it will

**Comment by Sc0rpi0n101** [comment]
Can you please check and confirm, so I can merge this?

**Comment by oscarbenjamin** [comment]
The API is
```
        >>> tf4.is_stable() is None   # Not enough info about the symbols to determine stability
```
Is there a way to obtain the conditions under which the system would be stable?

That's the kind of thing that a symbolic rather than numeric control theory library could actually be useful for in practice.

File: sympy/physics/control/lti.py [hunk 1]
```diff
@@ -355,10 +355,8 @@ def is_stable(self):
         >>> tf3 = TransferFunction(4, q*s - r, s)
         >>> tf3.is_stable()
         False
-
-        # not enough info about the symbols to determine stability
```
Comment (eric-wieser) [inline sympy/physics/control/lti.py:359]: I think
```
>>> # not enough info about the symbols to determine stability
```
would also work here, as would making this not be a comment at all and adding a trailing newline
```
Note this returns None if there is not enough info about the symbols to determine stability

```

**Review by eric-wieser — APPROVED** [review]

**Comment by eric-wieser** [comment]
> Is there a way to obtain the conditions under which the system would be stable?

Yes, that's just `And(*(re(p) < 0 for p in self.poles()))`.

**Comment by namannimmo10** [comment]
I think this PR is ready to merge.

**Comment by sylee957** [comment]
I'd expect improving this with other frameworks like https://en.wikipedia.org/wiki/Routh%E2%80%93Hurwitz_stability_criterion since they requires no explicit determination for roots, but only from its polynomial coefficients.

**Comment by namannimmo10** [comment]
> I'd expect improving this with other frameworks like https://en.wikipedia.org/wiki/Routh%E2%80%93Hurwitz_stability_criterion since they requires no explicit determination for roots, but only from its polynomial coefficients.

Have a look [here](https://github.com/ilayn/harold/pull/62).

**Pull request merged**